template/scripts/sample-script.js

```diff
@@ -3,7 +3,7 @@
 //
 // When running the script with `npx hardhat run <script>` you'll find the Hardhat
 // Runtime Environment's members available in the global scope.
-const hre = require("hardhat");
+const hre = require('hardhat');
```

Comment:
nit: since we only use `ethers`, may be we just import that?

Reply:
I think using `hardhat.ethers` instead of `ethers` lets us use utilities like `getContractFactory`. I could be wrong though.